.circleci/config.yml

```
@@ -1,7 +1,7 @@
 version: 2.1
 orbs:
   cypress: cypress-io/cypress@1.28.0
-  codecov: codecov/codecov@1.1.3
+  codecov: codecov/codecov@1.2.1
```

```suggestion
  codecov: codecov/codecov@1.2.2
```
Hopefully this version will work now